lib/job/arBaseJob.class.php

```diff
@@ -27,11 +27,15 @@
 
 class arBaseJob extends Net_Gearman_Job_Common
 {
-  private $requiredParameters = array();
+  // Required parameters for all jobs:
```

Comment:
It's not like you are introducing a list of items here. I'd just reword this with a simple paragraph.